.github/workflows/workflow.yaml

```diff
@@ -12,10 +12,13 @@ jobs:
     with:
       image_name: ${{ github.event.repository.name }}
       image_tag: 'latest'
-      quay_img_exp: 'never'
```

Comment:
Is this removed because it's default?

Reply:
Yes. I am doing my best to simplify the standard template, and all optional configurable inputs will live only in the docs. It is not necessary to have this in the template as "never" would be the default and wanted behavior.